### PR TITLE
Add kart kourse question

### PR DESCRIPTION
**Pull request by AngelOnFira**

### CODE REVIEW

File: questions/kart-course.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Kart Kourse
+
+## Question
+
+Welcome to **Kart Kourse**! Here is the problem statement:
+
+You are given a list of racers' names in race position order and a list of
+position changes. Your task is to write a function that returns the new order of
+racers after applying the position changes.
```
Comment (eros-mcguire) [inline questions/kart-course.md]:
```suggestion
You are given a list of racers' names in the position they start the race in and a list of
position changes. Your task is to write a function that returns the new order of
the racers after applying the position changes.
```

File: questions/kart-course.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+The first element in `position_changes` indicates that the racer in the second
+position (Luigi) advances two places. The second element indicates that the
+racer in the third position (Peach) advances one place. The third element
+indicates that the racer in the fourth position (Yoshi) moves down one place.
+The fourth element indicates that the racer in the fifth position (Toad) stays
+in place. The fifth element indicates that the racer in the first position
+(Mario) moves down two places.
+
```
Comment (marceljc3) [inline questions/kart-course.md:27]: indexes do not match?

**Review by marceljc3 — CHANGES_REQUESTED** [review]

File: questions/kart-course.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+## Input
+
+Here is an example input:
+
+```python
+racers = ["Mario", "Luigi", "Peach", "Yoshi", "Toad"]
+position_changes = [2, 1, -1, 0, -2]
+```
+
+The first element in `position_changes` indicates that the racer in the second
```
Comment (djoga24) [inline questions/kart-course.md:20]: Elements in the position_changes should align with the elements in the racers list. Unintuitive that they don't

File: questions/kart-course.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+Welcome to **Kart Kourse**! Here is the problem statement:
+
+You are given a list of racers' names in race position order and a list of
+position changes. Your task is to write a function that returns the new order of
+racers after applying the position changes.
+
+## Input
+
+Here is an example input:
```
Comment (l004p) [inline questions/kart-course.md]:
```suggestion
toes
```

File: questions/kart-course.md (hunk 2 above)
Comment (yufengliu15) [inline questions/kart-course.md:27]: explaning `position_changes` could be clarified a little better. For example: `position_changes[i]` represents the position for `racers[i+1]`, and the last element in `position_changes` is for `racers[0]`

File: questions/kart-course.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+position_changes = [2, 1, -1, 0, -2]
+```
+
+The first element in `position_changes` indicates that the racer in the second
+position (Luigi) advances two places. The second element indicates that the
+racer in the third position (Peach) advances one place. The third element
+indicates that the racer in the fourth position (Yoshi) moves down one place.
+The fourth element indicates that the racer in the fifth position (Toad) stays
+in place. The fifth element indicates that the racer in the first position
+(Mario) moves down two places.
```
Comment (djoga24) [inline questions/kart-course.md]:
```suggestion
The first element in `position_changes` indicates that the racer in the second
position (Mario) advances two places. The second element indicates that the
racer in the third position (Peach) advances one place. The third element
indicates that the racer in the fourth position (Yoshi) moves down one place.
The fourth element indicates that the racer in the fifth position (Toad) stays
in place. The fifth element indicates that the racer in the first position
(Mario) moves down two places.
```
The first element in `position_changes` indicates that the racer in the first
position (Mario) advances two places. The second element indicates that the
racer in the second position (Luigi) advances one place. The third element
indicates that the racer in the third position (Peach) moves down one place.
The fourth element indicates that the racer in the fourth position (Toad) stays
in place. etc

**Review by djoga24 — APPROVED** [review]

File: questions/kart-course.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+You are given a list of racers' names in the position they start the race in and a list of
+position changes. Your task is to write a function that returns the new order of
+the racers after applying the position changes.
+
+## Input
+
+toes
+
+```python
+racers = ["Mario", "Luigi", "Peach", "Yoshi", "Toad"]
```
Comment (eros-mcguire) [inline questions/kart-course.md]:
```suggestion
racers = ["John", "Forest", "Matthew", "Victor", "Jacc"]
```

**Review by marceljc3 — APPROVED** [review]
approved

**Pull request merged**